manon/variables/_form-help.scss

```diff
@@ -3,7 +3,7 @@ $form-help-container-padding: 0 2.5rem 0 0 !default;
 
 // Help button
 $form-help-button-top: 0 !default;
-$form-help-button-right: 0 !default;
+$form-help-button-right: -2rem !default;
```

Comment:
Using a negative value as a default might be confusing for theme builders later on. I would prefer to keep this as 0. I understand we want to create space for the help button. I would suggest to add a padding to the parent div instead. Which can be set to null and be filled by theme builders if they choose to do so.